src/framework/components/screen/component.js

```diff
@@ -286,19 +286,24 @@ class ScreenComponent extends Component {
     }
 
     /**
-     * Priority determines the order in which screens components are rendered. Priority is set into
-     * the top 8 bits of the drawOrder property in an element.
+     * Priority determines the order in which Screen components in the same layer are rendered.
+     * Number must be an integer between 0 and 255.
+     * Priority is set into the top 8 bits of the drawOrder property in an element.
      *
      * @type {number}
-     * @private
+     * @public
```

Comment:
You don't need this tag. It will be public by default, so you can delete it.

Reply:
Good call!